html/browsers/the-window-object/window-indexed-properties.html

```diff
@@ -20,12 +20,17 @@
 test(function() {
   window[0] = "foo";
   assert_throws(new TypeError(), () => Object.defineProperty(window, 0, { value: "bar" }))
+  assert_throws(new TypeError(), () => Object.defineProperty(window, 0, { get() { return "baz" } }))
+  assert_throws(new TypeError(), () => Object.defineProperty(window, 0, { set() { return "quz" } }))
```

Comment:
Well, it'd be nice to be consistent, but it no longer matters.

My above comment about asserting window[1] still stands though.